docs/source/diagnosis.md

```diff
@@ -0,0 +1,85 @@
+1. [Diagnosis introduction](#diagnosis-introduction)
+2. [Supported Feature Matrix](#supported-feature-matrix)
+3. [Get started](#get-started)
+4. [Example](#example)
+
+# Diagnosis introduction
+To validate the quality of the optimized model we should run diagnostics and check its accuracy and performance.
+There are 2 ways to diagnose a model with Intel® Neural Compressor. First is non-GUI mode that is described below and second is GUI mode with [Neural Insights](https://github.com/intel/neural-compressor/tree/master/neural_insights) component.
+
+The workflow is described in the diagram below. First we have to configure scripts with diagnosis, then run them and check diagnosis info in the terminal. Test if the result is satisfying and repeat the steps if needed.
+![workflow](./imgs/workflow.jpg)
+
+# Supported feature matrix
+<table class="center">
+    <thead>
+        <tr>
+            <th>Types</th>
+            <th>Diagnosis data</th>
+            <th>Framework</th>
+            <th>Backend</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan="2" align="center">Post-Training Static Quantization (PTQ)</td>
+            <td rowspan="2" align="center">weights and activations</td>
+            <td align="center">TensorFlow</td>
+            <td align="center"><a href="https://github.com/tensorflow/tensorflow">TensorFlow</a>/<a href="https://github.com/Intel-tensorflow/tensorflow">Intel TensorFlow</a></td>
+        </tr>
+        <tr>
+            <td align="center">ONNX Runtime</td>
+            <td align="center"><a href="https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/quantize.py">QLinearops/QDQ</a></td>
+        </tr>
+    </tbody>
+</table>
+
+# Get started 
+First you need to install Intel® Neural Compressor.
```

Comment:
In get started part, let show the code change we should made in a scripts, like this： 
![image](https://github.com/intel/neural-compressor/assets/51692656/69598de6-4d48-4991-bbbb-e04902500e55)
Add both quantization and benchmark.